### ISSUE TITLE
Inconsistent parameter names for two sites from the same project (NRS)

### ISSUE THREAD

**Issue by Abulos** [aodn/aodn-portal#2927]
This issue tracker is only for AODN Portal issues.

### Steps to reproduce the issue
1. Download .csv files for NRS project IMOS - Combined Biogeochemical parameters (reference stations) PHB and KAI ie IMOS_-_Combined_Biogeochemical_parameters_(reference_stations)-Chemical_parameters_only.csv

### Expected behaviour
Tables of data from two sites in the same project should have consistent parameter names

### Actual behaviour
KAI has parameter name: SampleDate_UTC
PHB has parameter name: TripDate_UTC

KAI has parameter name: Phosphate_umoL
PHB has parameter name: Phosphate_umolL

### Screenshots.
NA

### What version of the aodn portal are you using?
No idea

### Which browser are you using?
Firefox 91.11.0esr (64-bit)

### Any extra information that might be useful in the debugging process.
This is normally the contents of a `debug.log` or `config.log` file. Raw text or a link to a pastebin type site are preferred.

**Comment by leonardolaiolo** [comment]
Hi @Abulos 
I cannot replicate the behaviour you explained above. On the other hand, we are making few adjustments to this product and what you described (SampleDate_UTC vs TripDate_UTC and Phosphate_umoL vs Phosphate_umolL) are changes we implemented at the end of last week. Thus, if you download again `IMOS_-Combined_Biogeochemical_parameters(reference_stations)-Chemical_parameters_only.csv` for both KAI anf PHB stations (together or independently) it should include TripDate_UTC and Phosphate_umolL for both stations. Let us know if you have any issue and sorry for the recent changes - we released this product recently so we are still making few adjustments.
